.changeset/wise-carrots-float.md

```diff
@@ -0,0 +1,15 @@
+---
+'astro': minor
+---
+
+Removes the experimental `contentCollectionsCache` introduced in `3.5.0`.
+
+To migrate, remove the flag from your Astro config:
+
+```diff
+export default defineConfig({
+    experimental: {
+-        contentCollectionsCache: true
+    }
+})
+```
```

Comment:
The replacement for this is migrating to the Content Layer, right? It'd be nice to include that reasoning in the changelog so users have all the context.